content/source/docs/enterprise-beta/api/o-auth-tokens.html.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: enterprise2
+page_title: "OAuth Tokens - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-o-auth-tokens"
+---
+
+# OAuth Tokens
+
+-> **Note**: These API endpoints are in beta and may be subject to change.
```

Comment:
style: "may" is ambiguous, since it means either "is allowed to" or "might possibly." Also redundant in this case. I suggest "…are in beta and are subject to change."

Reply:
Ah, I see that's a carry-over from prior API pages; no need to change all of them on this PR, but we should when it's convenient.